packages/examples/nodejs-interactive/src/constants.ts

```diff
@@ -0,0 +1,43 @@
+export interface RpcRequest {
+  method: string;
+  params: any[];
+}
+
+export interface Chain {
+  chainId: string;
+  chainName: string;
+  blockExplorerUrls: string[];
+  nativeCurrency: { symbol: string, decimals: number };
+  rpcUrls: string[];
+}
+
+export const chains = {
```

Comment:
You should remove `as Chain`
Instead declare it this way:
export const chains: {[name: string]: Chain} = { ... }

Reply:
Done, thank you!